bindings/node/lib/bindings/pre-tokenizers.test.ts

```diff
@@ -44,6 +45,13 @@ describe("punctuationPreTokenizer", () => {
   });
 });
 
+describe("splitPreTokenizer", () => {
```

Comment:
This test fails -> not sure what's going on. I didn't manage to see a good error message :-/

Reply:
How did you compile the node bindings? Did you run `npm run dev-rs`? I'm asking because it seems fine for the CI, and on my local install too.

Reply:
ah yeah I didn't - didn't know how to compile the bindings, now I know :-)